tide_station_locator.module

```diff
@@ -30,28 +31,62 @@ function tide_station_locator_get_api_data($delta = FALSE) {
 /**
  * Function to migrate stations.
  */
-function tide_station_locator_migrate_stations($options) {
+function tide_station_locator_migrate_stations() {
 
   try {
     // Connect to API and download the latest station JSON and migrate.
-    tide_station_locator_get_api_data($options['delta']);
-    if ($options['execute-dependencies'] == TRUE) {
-      $dependent_migrations =
-        [
-          'station_accessibility_terms',
-          'station_speciality_services_facility_terms',
-          'station_state_terms',
-        ];
-      foreach ($dependent_migrations as $migration_ids) {
-        _tide_station_locator_migrate($migration_ids);
-      }
-    }
+    tide_station_locator_get_api_data();
     _tide_station_locator_migrate('station_nodes');
   }
   catch (Exception $e) {
     // Something went wrong somewhere.
     watchdog_exception('tide_station_locator', $e);
   }
+
+  // Set the correct moderation of stations based on dates.
+  _tide_station_locator_content_moderator();
+}
+
+/**
+ * Helper method to set the correct moderation based on dates.
+ */
+function _tide_station_locator_content_moderator() {
```

Comment:
hey @g2410g , why do we need _tide_station_locator_content_moderator() here? Couldn’t we use a process plugin to set moderation_state instead?

Reply:
@vincent-gao Updated the code.